charts/starship/templates/agent/daemonset.yaml

```diff
@@ -1,21 +1,22 @@
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: {{ include "tricorder.fullname" . }}-tricorder-agent
+  name: tricorder
   namespace: {{ .Release.Namespace }}
-  labels:
```

Comment:
helm uninstall would not works well if loss `helm.sh/chart:xxx` label , I think.